joomla/joomla-cms
# [4.0] Fix show password icon in login forms #36104

Pull Request for Issue #36104.

### Summary of Changes
         In short, This PR makes the show password icon to icon-eye-slash, when the password is not visible and the show 
         password icon changes to icon-eye, when the password is visible.

### Testing Instructions
        see login form on
        1. http://localhost/joomla-cms/administrator/ .
        2. http://localhost/joomla-cms/.
        3. http://localhost/joomla-cms/index.php/login (after installing sample data).
        
         
 

### Actual result BEFORE applying this Pull Request
        
![image](https://user-images.githubusercontent.com/60544908/144714386-d5aa57ce-dfb7-4420-9904-e33efe2a645b.png)
![text wrong](https://user-images.githubusercontent.com/60544908/144718674-b4a7121d-989a-481e-9019-0c961539a146.PNG)



      

 ### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/60544908/144714393-2abb4dee-db5c-4200-8753-ca5200fc9b10.png)
![text2](https://user-images.githubusercontent.com/60544908/144718667-2372af73-fe49-4c57-a614-747103e2c2b4.PNG)



### Documentation Changes Required
        None.
### Note
         This PR made changes on 
        1. http://localhost/joomla-cms/administrator/ login form.
        2. http://localhost/joomla-cms/ login form.
        3. http://localhost/joomla-cms/index.php/login login form.
